### DISCUSSION TITLE
Expanding SNSD into MNSD -- Is it possible?

### DISCUSSION THREAD

**Discussion by wizpresso-steve-cy-fan** [minio/minio#16923] [Q&A] [answered]
We want to deploy Minio on our NAS servers first and then we will also try to gradually add our existing servers into the matrix after evaluations and considerations, so while we can just run Minio SNSD using the Docker on our NAS, but we are not sure if we can expand it to multiple servers afterwards, because under SNSD it has zero-parity...

**Comment by harshavardhana** [comment, marked as answer]
yes @wizpresso-steve-cy-fan https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html

**Comment by wizpresso-steve-cy-fan** [comment]
@harshavardhana No. We are going to run a clean SNSD deployment first, and we want to incrementally move into MNMD...

**Reply by harshavardhana** [reply]
That is not possible what we recommend you do something like add a new pool and then decommissioning the 1 disk setup.
